src/apps/experimental/routes/user/display/index.tsx

```diff
@@ -0,0 +1,107 @@
+import Button from '@mui/material/Button';
+import { SelectChangeEvent } from '@mui/material/Select';
+import Stack from '@mui/material/Stack';
+import React, { useCallback, useEffect } from 'react';
+
+import viewManager from 'components/viewManager/viewManager';
+import globalize from 'scripts/globalize';
+import theme from 'themes/theme';
+import { DisplayPreferences } from './DisplayPreferences';
+import { ItemDetailPreferences } from './ItemDetailPreferences';
+import { LibraryPreferences } from './LibraryPreferences';
+import { LocalizationPreferences } from './LocalizationPreferences';
+import { NextUpPreferences } from './NextUpPreferences';
+import { useDisplaySettingForm } from './hooks/useDisplaySettingForm';
+import { DisplaySettingsValues } from './types';
+import LoadingComponent from 'components/loading/LoadingComponent';
+
+export default function UserDisplayPreferences() {
+    useEffect(() => {
+        /**
+         * NOTE: LEGACY SUPPORT
+         *
+         * Hide any legacy views which were shown when this component mounts. The legacy view manager automatically
+         * hides the previous view when a new view is shown. Since this is a React page not managed by the legacy view
+         * manager, we need to manually hide the previous view to prevent it from remaining visible on the screen.
+         */
+        viewManager.hideView();
```

Comment:
Using the `<Page>` component should handle all the unpleasantness of dealing with the legacy view code for you.

https://github.com/jellyfin/jellyfin-web/blob/master/src/components/Page.tsx

Reply:
Oh nice, I hadn't noticed that. I'll take a look at integrating it this evening.

Reply:
You should be able to remove this effect now since it is handled in the `<Page>`component.

Reply:
Oof, hit by that post-work brain fog and forgot to remove the effect. 😅